Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -621,19 +765,49 @@ extension SwitchCaseSyntax {
   
 }
 
-private let autoSynthesizingProtocolTypes: Set<String> = [
+let equatableProtocolNames: Set<String> = [
```

Comment:
Explicit `private` access control is recommended. This convention is learned from the Swift Standard Library.